Client/MainWindowViewModel.cs

```diff
@@ -5,18 +5,22 @@
 using System.Windows.Controls;
 using System.Windows.Data;
 using ToDoLib;
-using System.Collections.ObjectModel;
 using System.Windows;
 using System.Windows.Input;
 using System.IO;
 using ColorFont;
 using CommonExtensions;
-using Microsoft.Win32;
 using System.Reflection;
 using System.Diagnostics;
 using System.ComponentModel;
 using System.Text.RegularExpressions;
 using System.Globalization;
+using Clipboard = System.Windows.Clipboard;
+using Control = System.Windows.Controls.Control;
+using KeyEventArgs = System.Windows.Input.KeyEventArgs;
+using MessageBox = System.Windows.MessageBox;
+using OpenFileDialog = Microsoft.Win32.OpenFileDialog;
+using SaveFileDialog = Microsoft.Win32.SaveFileDialog;
```

Comment:
It's not immediately obvious to me why we need to import these winforms components?

Reply:
I removed those imports.